pmlc/dialect/pxa/transforms/autotile.cc

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020, Intel Corporation
+
+#include "pmlc/dialect/pxa/transforms/autotile.h"
+
+namespace pmlc::dialect::pxa {
+
+std::vector<int64_t> PowerOfTwoGenerator(int64_t range) {
+  std::vector<int64_t> out;
+  for (int64_t r = 1; r <= range; r *= 2) {
+    out.push_back(r);
+  }
+  return out;
+}
+
+std::vector<int64_t> EvenDivisionGenerator(int64_t range) {
```

Comment:
Could you move these generally useful sorts of things to `pmlc/util` somewhere and export them?  (I was actually about to write this for another project; I'd rather just call your version.  :-)

Reply:
If this winds up in upstream MLIR, then it would probably belong in `mlir/Support/Generators.h` or some such.

Reply:
I think we can totally move the, but I'm not sure where I'd put them right now, so leaving it.